variables.tf

```diff
@@ -55,6 +43,57 @@ variable "access_tags" {
   }
 }
 
+##############################################################################
+# Naming Variables
+##############################################################################
+
+variable "prefix" {
+  description = "The value that you would like to prefix to the name of the resources provisioned by this module. Explicitly set to null if you do not wish to use a prefix."
+  type        = string
+  default     = null
+}
+
+variable "name" {
+  description = "If `create_vpc` is true, this string names both the VPC and associated resources. If false, it names only the associated resources."
```

Comment:
Used for the naming of the VPC (if `create_vpc` is set to true), as well as in the naming for any resources created inside the VPC (unless using one of the optional variables for explicit control over naming).